lib/linguist/languages.yml

```
@@ -1399,6 +1399,17 @@ Cycript:
   codemirror_mode: javascript
   codemirror_mime_type: text/javascript
   language_id: 78
+Cylc:
+  type: programming
```

This should be data as is INI into which you're grouping it.

---

Thanks:

```suggestion
  type: data
```

The syntax is INI-like, but it describes, an executable program, let me know if there's a more appropriate grouping.

---

My comment was only based on you grouping it with INI. Grouping isn't necessary. 

I know nothing about Cylc but from a quick view of the docs it appears these files describe workflows so it's really up to you if you'd consider this an actual programming language or data. We have other workflow-like languages which are considered data and others programming.

---

Yep that's right, so long as it makes sense in Linguist.